Makefile

```diff
@@ -25,25 +25,22 @@ $(OUTPUT_TAGS).md: index_tags.md tagging.md matroska_tagging_registry.md tagging
 	cat $^ > $@
 
 %.xml: %.md
-	mmark -xml2 -page $< > $@
+	mmark -xml2 -page $< | awk '/<?rfc toc=/ && !modif { printf("<?rfc tocdepth=\"12\"?>\n"); modif=1 } {print}' > $@
```

Comment:
I see a value of 12 for tocdepth here, but the example I saw on gist goes six elements deep. Will this patch produce a tocdepth of 6? I can't test right now unfortunately as I'm about to leave work.

Reply:
Ya it fails to use depths above 6. I think this may be related to html which only defines down to <h6>. h7 and so on are invalid. We could squash the other elements back to h6. Possibly we could elevate the whole section since it already starts at h3.